app/controllers/users_controller.rb

```diff
@@ -2,30 +2,34 @@ class UsersController < ApplicationController
   before_action :authenticate_user!
 
   def edit
-    @billing = Billing.where(user: current_user).first_or_initialize
-    @shipping = Shipping.where(user: current_user).first_or_initialize
+    @billing = Billing.where(user: current_user).first_or_create
```

Comment:
Why do you create objects here?

Reply:
Because with `first_or_initialize` it throws an exeption